apps/ledger-live-mobile/src/screens/Onboarding/steps/setupDevice/scenes/RestoreWithProtect.tsx

```diff
@@ -46,20 +43,19 @@ const Next = ({ onNext }: { onNext: () => void }) => {
   const restoreInfoDrawer =
     servicesConfig?.params?.onboardingRestore?.restoreInfoDrawer || {};
 
-  const manualStepsURI = restoreInfoDrawer?.manualStepsURI;
+  const updateFirmwareLink =
```

Comment:
This URL needs to be put in the feature flag, or the config/urls file ( `import { urls } from "../config/urls";` )